lib/src/components/NxCheckbox/NxCheckbox.scss

```diff
@@ -21,24 +21,49 @@ $-width: 16px;
     @include nx-focus-border-helpers.radio-checkbox-focus-border($-height, $-width);
 
     border-radius: 4px;
-    left: -5px;
-    top: -5px;
+    left: -4px;
+    top: -4px;
   }
 
   .fa-check {
-    color: var(--nx-swatch-white);
+    color: var(--nx-color-component-background);
     font-size: var(--nx-font-size-s);
-    vertical-align: 4px;
+    vertical-align: 3px;
+    margin-left: 1px;
```

Comment:
These can be avoided by turning the parent into an inline flex container with `align-items` and `justify-content` both set to `center`.